cppapi/server/eventsupplier.h

```diff
@@ -247,6 +247,20 @@ public :
 	TANGO_IMP_EXP static ZmqEventSupplier *create(Util *);
 	virtual ~ZmqEventSupplier();
 
+
+    class Future
+    {
+    public:
+    	Future();
+    	void set();
+    	void wait();
+    	void reset();
+    	bool is_ready();
```

Comment:
You actually never call these two methods, if you do not plan to use them, consider removing.